test/integration/view_pursuits_test.rb

```diff
@@ -0,0 +1,33 @@
+require 'test_helper'
+
+class ViewPursuitsTest < ActionDispatch::IntegrationTest
+  def create_pursuits(num)
+    num.times do |i|
+      i += 1
+      Pursuit.create(name: "Hiking the Alps #{i}", description: "Go hike the alps! #{i}", price: 1000+i) #rou, activity_id: i, location_id: i)
+    end
+  end
+
+  test 'visitor can see pursuits' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.